circus-train-integration-tests/src/test/data/com/hotels/bdp/circustrain/integration/CircusTrainHdfsHdfsIntegrationTest/partitioned-single-table-avro-schema-metadata-update.yml

```diff
@@ -7,5 +7,6 @@ table-replications:
       partition-limit: 100
     replica-table:
       table-name: ct_table_p_managed_copy
-avro-serde-options:
-  base-url: ${circus-train-runner.replica-warehouse-uri}${circus-train-runner.database-name}/
\ No newline at end of file
+transform-options:
+  avro-serde-options:
+    base-url: ${circus-train-runner.replica-warehouse-uri}${circus-train-runner.database-name}/
```

Comment:
missing newlines in some of the YML files

Reply:
Yeah that's how I found them, but I'll add them in